external-crates/move/crates/move-analyzer/editors/code/package.json

```diff
@@ -44,7 +44,12 @@
       },
       {
         "command": "move.test",
-        "title": "Test a Move package",
+        "title": "Run Move package test",
+        "category": "Move"
+      },
+      {
+        "command": "move.trace",
+        "title": "Trace Move package execution",
```

Comment:
Could we say 'test' instead of 'package'?

Reply:
Sure! Done!